distsql/distsql.go

```diff
@@ -197,7 +240,7 @@ func (pr *partialResult) Close() error {
 
 // SelectDAG sends a DAG request, returns SelectResult.
 // In kvReq, KeyRanges is required, Concurrency/KeepOrder/Desc/IsolationLevel/Priority are optional.
-func SelectDAG(ctx goctx.Context, client kv.Client, kvReq *kv.Request, colLen int) (SelectResult, error) {
+func SelectDAG(ctx goctx.Context, client kv.Client, kvReq *kv.Request, fieldTypes []*types.FieldType, loc *time.Location) (SelectResult, error) {
```

Comment:
We can get `*time.Location` from `ctx`, so it's unnecessary to be a extra argument.

Reply:
It's `goctx.Context`.